dropwizard-logging/src/test/java/io/dropwizard/logging/common/TestPatternLayoutFactory.java

```diff
@@ -38,7 +38,11 @@ protected Map<String,Supplier<DynamicConverter>> getDefaultConverterSupplierMap(
             return Collections.emptyMap();
         }
 
+        /**
+         * @deprecated Use {@link #getDefaultConverterSupplierMap()} instead
+         */
         @Override
+        @Deprecated
```

Comment:
should we add a "since" in here? (same comment below)

Reply:
Actually on second thoughts I'm not sure the javadoc is adding anything - this a test class and the other one is a private inner class. I've taken it out.